app/services/directions_service.rb

```diff
@@ -0,0 +1,12 @@
+class DirectionsService
+  def self.conn
+    Faraday.new(url: "http://www.mapquestapi.com") do |faraday|
+      faraday.params[:key] = ENV['key']
```

Comment:
I would probably name the api key something like `mapquest_api_key` to make it a little clearer to other devs on the team what this is/where it's coming from.